docker_entrypoint.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
Well if we want to move it there, it would make sense to move the Dockerfile there as well as some other teams are doing.
The problem is that it might confuse people as they would expect a Dockerfile in the main folder.
The current docker folder is for dev only atm and I don't want to mix things up.

We should decide what we want/like and refactor it, in another PR maybe.